src/charts/LineChart.js

```diff
@@ -118,14 +118,27 @@ function LineChart( properties ) {
           } );
 
     // Add the X Axis
-    svg.append('g')
+    // Add the X Axis
+    var xAxis = svg.append('g')
       .classed('axis axis__x', true)
       .attr('transform', 'translate(0,' + height + ')')
       .call( d3.axisBottom( x )
-         .tickFormat( d3.timeFormat( '%b %Y' ) )
-      )
-      .selectAll( 'text' )
-        .attr( 'dy', '15px' );
+         .tickFormat( function( d ) { return ''; }  )
+      );
+
+    xAxis.selectAll( 'g' )
+        .append( 'text' )
+          .style( 'text-anchor', 'middle' )
+          .attr( 'y', 25 )
+          .text( function( d) { return getMonth( d ); } )
```

Comment:
testing this with npm link. JS error here because getMonth is undefined in this file